lib/ClimaLandSimulations/src/fluxnet_simulation.jl

```diff
@@ -0,0 +1,350 @@
+            sc = sc,
+            pc = pc,
+            Vcmax25 = Vcmax25,
+            Γstar25 = Γstar25,
+            Kc25 = Kc25,
+            Ko25 = Ko25,
+            To = To,
+            ΔHkc = ΔHkc,
+            ΔHko = ΔHko,
+            ΔHVcmax = ΔHVcmax,
+            ΔHΓstar = ΔHΓstar,
+            ΔHJmax = ΔHJmax,
+            ΔHRd = ΔHRd,
+        )
+    )
+    # Set up plant hydraulics
+    ai_parameterization = PrescribedSiteAreaIndex{FT}(LAIfunction, SAI, RAI)
+
+    function root_distribution(z::T; rooting_depth = rooting_depth) where {T}
+        return T(1.0 / rooting_depth) * exp(z / T(rooting_depth)) # 1/m
+    end
+
+    plant_hydraulics_ps = PlantHydraulics.PlantHydraulicsParameters(;
+        ai_parameterization = ai_parameterization,
+        ν = plant_ν,
+        S_s = plant_S_s,
+        root_distribution = root_distribution,
+        conductivity_model = conductivity_model,
+        retention_model = retention_model,
+    )
+    plant_hydraulics_args = (
+        parameters = plant_hydraulics_ps,
+        n_stem = n_stem,
+        n_leaf = n_leaf,
+        compartment_midpoints = compartment_midpoints,
+        compartment_surfaces = compartment_surfaces,
+    )
+
+    energy_args = (parameters = Canopy.BigLeafEnergyParameters{FT}(ac_canopy),)
+
+    # Canopy component args
+    canopy_component_args = (;
+        autotrophic_respiration = autotrophic_respiration_args,
+        radiative_transfer = radiative_transfer_args,
+        photosynthesis = photosynthesis_args,
+        conductance = conductance_args,
+        hydraulics = plant_hydraulics_args,
+        energy = energy_args,
+    )
+
+    # Other info needed
+    shared_params = SharedCanopyParameters{FT, typeof(earth_param_set)}(
+        z0_m,
+        z0_b,
+        earth_param_set,
+    )
+
+    canopy_model_args = (; parameters = shared_params, domain = canopy_domain)
+
+    # Integrated plant hydraulics and soil model
+    land_input = (atmos = atmos, radiation = radiation)
+    land = SoilCanopyModel{FT}(;
+        soilco2_type = soilco2_type,
+        soilco2_args = soilco2_args,
+        land_args = land_input,
+        soil_model_type = soil_model_type,
+        soil_args = soil_args,
+        canopy_component_types = canopy_component_types,
+        canopy_component_args = canopy_component_args,
+        canopy_model_args = canopy_model_args,
+    )
+    Y, p, cds = initialize(land)
+    exp_tendency! = make_exp_tendency(land)
+
+    #Initial conditions
+    Y.soil.ϑ_l =
+        drivers.SWC.status != absent ?
+        drivers.SWC.values[1 + Int(round(t0 / DATA_DT))] : soil_ν / 2 # Get soil water content at t0
+    # Both data and simulation are reference to 2005-01-01-00 (LOCAL)
+    # or 2005-01-01-06 (UTC)
+    Y.soil.θ_i = FT(0.0)
+    T_0 =
+        drivers.TS.status != absent ?
+        drivers.TS.values[1 + Int(round(t0 / DATA_DT))] :
+        drivers.TA.values[1 + Int(round(t0 / DATA_DT))] + 40# Get soil temperature at t0

```

Comment:
why plus 40?

Reply:
uh I have no idea... I copy pasted this https://github.com/CliMA/ClimaLSM.jl/blob/main/experiments/integrated/fluxnet/run_fluxnet.jl#L279 @Espeer5 do you know?